packages/components/src/passengerCard/PassengerCard.js

```diff
@@ -135,9 +135,13 @@ class PassengerCard extends React.Component<Props> {
                 {bags &&
                   bags.map((bag, idx) => (
                     <BagInformation
-                      key={`${bag.dimensions}, ${idx}`}
+                      key={`${
+                        bag.dimensions ? `${bag.dimensions},` : ''
+                      }, ${idx}`}
                       count={bag.quantity}
-                      type={`${bag.dimensions}, ${bag.weight}`}
+                      type={`${
```

Comment:
It is little bit hard to read. What about at least extracting it to top of the render function next to quite similar `passengerWithTitle`?

Reply:
Done!

Reply:
Sorry I didn't noticed that it is actually nested inside `map`. In this case we can probably move this new parser even slightly up out of render function. What do you think?

Reply:
Done!